hackathon_site/dashboard/frontend/src/pages/TeamDetail/TeamDetail.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import styles from "./TeamDetail.module.scss";
```

Comment:
Since you're not using the styling file anymore, you can remove the import. You can keep the file until the page is done in case we need it in the future.